### DISCUSSION TITLE
FEP cleaning tool

### DISCUSSION THREAD

**Discussion by trohnjavolta** [sn4k3/UVtools#170] [Ideas]
Hi, this video gave me a small idea that I think could be handy: https://www.youtube.com/watch?v=3L59JGGJOjs

I don't have such a tank clean option on my printer but I think it just runs a quick print where just the complete screen is exposed for one (maybe more?) layers so one can peel off the whole layer including any residual cured resin from a failed print.

No idea if this is already possible somehow in uvtools, but I think a one click tool entry that creates such a tank/fep cleaning file would be great. I then could store that on my printers internal storage.
Put in a small piece of cured resin on the edege of the fep like from supports, run the fep cleaning file and then pull on the corner piece to lift out the whole layer.
So the cleaning file of course shouldn't do any z movement.

What do you think, is that possible, usefull?

**Comment by sn4k3** [comment]
You can do that in UVtools, but "So the cleaning file of course shouldn't do any z movement." is not possible for most of the printers when you print a file, since the firmware will follow the print path and enforce some values. Only gcode printers can do it. The "On LCD" function we see is a special procedure outside print function

Here what you must do:

1) Use any file for your printer on UVtools
2) Remove all layers but leave the first (layer 0)
3) Use pixel editor with the biggest brush size and paint all layer White
4) Tune exposure time that you fell ok
5) Print it without the build plate

Then you can always use that file to do this action

**Comment by X3msnake** [comment]
For Chitu based printer, custom chitu boards like phrozen sonic mini
included, you can just create a txt file and type the following

M7004 I1 T1
It starts the LED with a open screen but afik the only way to stop it is to
go to the tools and press stop

there is also this universal UI for all printers that have the cleanvat
button
https://www.facebook.com/groups/chitusystems/permalink/854624515322513


Tiago Conceição ***@***.***> escreveu no dia quinta,
18/03/2021 à(s) 16:08:

> You can do that in UVtools, but "So the cleaning file of course shouldn't
> do any z movement." is not possible for most of the printers when you print
> a file since the firmware will follow the print path and enforce some
> values. Only gcode printers can do it. The LCD function we see is a special
> procedure outside print function
>
> Here what you must do:
>
>    1. Use any file for your printer on UVtools
>    2. Remove all layers but leave the first (layer 0)
>    3. Use pixel editor with the biggest brush size and paint all layer
>    White
>    4. Tune exposure time that you fell ok
>    5. Print it without the build plate
>
> Then you can always use that file to do this action
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sn4k3/UVtools/discussions/170#discussioncomment-499228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUR56R6FTOSWKNSWER3DH3TEIQRDANCNFSM4ZM462SA>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva

**Comment by sn4k3** [comment]
https://github.com/sn4k3/UVtools/blob/master/UVtools.ScriptSample/ScriptVATClean.cs

**Comment by d-kozminihs** [comment]
In some printers there is a function called "Detection" or "Exposure". All you need to do is level the platform and strat the function to expose the whole screen. Does not work with some printers (like Original Photon) since they expose only part of the screen

**Comment by X3msnake** [comment]
It works on Original Photon. You only need a ui with that function. You can
either download it from chitusystems or get the latest high contrast UI
from photonsters that comes with the clean vat button/function

d-kozminihs ***@***.***> escreveu no dia segunda, 26/07/2021
à(s) 15:15:

> In some printers there is a function called "Detection" or "Exposure". All
> you need to do is level the platform and strat the function to expose the
> whole screen. Does not work with some printers (like Original Photon) since
> they expose only part of the screen
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sn4k3/UVtools/discussions/170#discussioncomment-1051509>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACUR56QJFXQSNVTCKOBDOM3TZVUWNANCNFSM4ZM462SA>
> .
>


-- 
Com os melhores cumprimentos,
Vinicius Silva
